### PR TITLE
talosctl: Add version 1.3.6

### PR DESCRIPTION
**Pull request by dedene**
Adds talosctl version 1.3.6.
Closes https://github.com/siderolabs/talos/issues/5458

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/talosctl.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+    "version": "1.3.6",
+    "description": "CLI for out-of-band management of Kubernetes nodes created by Talos<",
+    "homepage": "https://talos.dev",
+    "license": " MPL-2.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/siderolabs/talos/releases/download/v1.3.6/talosctl-windows-amd64.exe",
```
Comment (rashil2000) [inline bucket/talosctl.json:8]:
```suggestion
            "url": "https://github.com/siderolabs/talos/releases/download/v1.3.6/talosctl-windows-amd64.exe#/talosctl.exe",
```
You can rename the file in the URL itself

File: bucket/talosctl.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    },
+    "pre_install": "Get-ChildItem \"$dir\\talosctl-windows-amd64.exe\" | Rename-Item -NewName 'talosctl.exe'",
+    "bin": "talosctl.exe",
+    "checkver": {
+        "github": "https://github.com/siderolabs/talos"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/siderolabs/talos/releases/download/v$version/talosctl-windows-amd64.exe"
```
Comment (rashil2000) [inline bucket/talosctl.json:20]:
```suggestion
                "url": "https://github.com/siderolabs/talos/releases/download/v$version/talosctl-windows-amd64.exe#/talosctl.exe"
```

File: bucket/talosctl.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    "description": "CLI for out-of-band management of Kubernetes nodes created by Talos<",
+    "homepage": "https://talos.dev",
+    "license": " MPL-2.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/siderolabs/talos/releases/download/v1.3.6/talosctl-windows-amd64.exe",
+            "hash": "6a922d0516d213e6ac13084aa4fb78f7d6f81b7885060b8383256fe80980d751"
+        }
+    },
+    "pre_install": "Get-ChildItem \"$dir\\talosctl-windows-amd64.exe\" | Rename-Item -NewName 'talosctl.exe'",
```
Comment (rashil2000) [inline bucket/talosctl.json:12]:
```suggestion
```

**Comment by dedene** [comment]
@rashil2000 Awesome, I didn't know I could do that inline in the download url. Thanks for the suggestion, I changed it!

**Pull request merged**